.github/workflows/integration-test.yml

```diff
@@ -1170,3 +1170,23 @@ jobs:
           timeout_minutes: 8
           retry_wait_seconds: 5
           command: cd terraform/stress && terraform destroy --auto-approve
+          
+  E2E_EKS_Test:
```

Comment:
nit: Match the naming style of other jobs in the workflow for this and the EC2 job as well

Reply:
Ok. I did that because the title looks Illegible due to all the capitalization, but not a big deal.